docs/components/plugins/PlatformsSection.tsx

```diff
@@ -47,7 +41,7 @@ function getInfo(isSupported: IsSupported, { title }: Platform) {
   } else if (typeof isSupported === 'object') {
     return {
       children: (
-        <a css={STYLES_LINK} target="_blank" href={isSupported.pending}>
+        <a css={STYLES_LINK} target="_blank" href="#">
```

Comment:
I'm not sure if you intended to change the `href` to `#`, that breaks the "PR Pending" links on [pages like this one](http://cedrics-expo.s3-website.eu-central-1.amazonaws.com/versions/v40.0.0/sdk/amplitude/).